server/lib/schema/index.ts

```diff
@@ -2204,14 +2204,16 @@ const schema = new GraphQLSchema({
                     );
                   }
 
-                  await tree.$add("oracle", user, transaction);
+                  await tree.$add("oracle", user, { transaction });
 
                   const oracleRelation: UserTreeOracleRelation | null = await UserTreeOracleRelation.findOne(
                     {
                       where: {
                         UserId: user.id,
                         TreeId: tree.id,
                       },
+
+                      transaction,
```

Comment:
It looks like the field expects a `FindOptions` object, of which `transaction` is one property.  (Specifically, `FindOptions` is an interface that extends `QueryOptions`, which extends `Transactionable`, which has a `transaction` property). So I'd expect this to need to be surrounded by curly braces as well.

Reply:
Somewhat disappointed TypeScript didn't catch the `transaction type errors I made in the past PR. I wonder if there is a stricter check we could use. It shouldn't have accepted a `Transaction` object when it was looking for a `FindOptions` object...